easybuild/easyconfigs/q/QBOX/QBOX-r140b_fix.patch

```diff
@@ -0,0 +1,139 @@
+-      SpeciesHandler.C XCPotential.C LDAFunctional.C PBEFunctional.C PBESolFunctional.C \
+-      PBERevFunctional.C BLYPFunctional.C NonLocalPotential.C SampleReader.C SampleWriter.C \
+-      StructuredDocumentHandler.C SampleHandler.C AtomSetHandler.C WavefunctionHandler.C \
++
++ PLT=LINUX_X86_64_GNU_SALOMON
++#-------------------------------------------------------------------------------
++ #FFTWDIR=/apps/all/FFTW/2.1.5-gompi-2015g
++ FFTWDIR=/apps/all/FFTW/3.3.4-gompi-2015g
++ FFTWLIB=$(FFTWDIR)/lib/libfftw.a
++# BLASDIR=$(HOME)/software/blas/blas-linux_x86-64
++# LAPACKDIR=$(HOME)/software/lapack/lapack-linux_x86-64
++# SCALAPACK_DIR = /apps/all/ScaLAPACK/2.0.2-gompi-2015b-OpenBLAS-0.2.14-LAPACK-3.5.0
++ SCALAPACK_DIR = /apps/all/ScaLAPACK/2.0.2-gompi-2015g-OpenBLAS-0.2.14-LAPACK-3.5.0
++ SCALAPACKLIB  = $(SCALAPACK_DIR)/lib/libscalapack.a
++ #BLASDIR = /apps/all/OpenBLAS/0.2.14-LAPACK-3.5.0/lib
++ BLASDIR = /apps/all/OpenBLAS/0.2.14-GNU-4.9.3-2.25-LAPACK-3.5.0
++ 
++# CC=gcc
++ #CFLAGS = -O3 
++ 
++ CXX=mpic++
++ LD=$(CXX)
++
++ DFLAGS +=  -DUSE_CSTDIO_LFS -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64
++ 
++	 INCLUDE = -I$(FFTWDIR)/include
++	 
++	 CXXFLAGS= -g -O3 -DUSE_MPI -DSCALAPACK -DADD_ -D$(PLT) $(INCLUDE) $(DFLAGS)
++
++	# LIBPATH = -L$(FFTWDIR) -L$(BLASDIR) -L$(LAPACKDIR)
++	# LIBS =  $(SCALAPACKLIB) -lgfortran -lfftw -lblas -llapack 
++	 LIBPATH = -L$(FFTWDIR) -L$(BLASDIR)
++	 #LIBS =  $(SCALAPACKLIB) -lfftw -omp -openmp -lmkl_core -lmkl_gnu_thread -lmkl_gf_lp64
++	 LIBS =  $(SCALAPACKLIB) -lfftw3 -fopenmp -lopenblas -lgfortran
++
++	# LDFLAGS = $(LIBPATH) $(LIBS) -Wl,-rpath,/usr/local/tools/mkl-8.1.1.004/lib
++	 LDFLAGS = $(LIBPATH) $(LIBS) 
++
++	TESTEXECS=testMatrix testFTGrid testGridFunction testBasis \
++	testBlacsContext testSlaterDet testEnergyFunctional testSample \
++	testChargeDensity  testFourierTransform testSpecies testContext \
++	testEigenSolvers
++
++	QBSRC=AtomSet.C Atom.C SymmetrySet.C Symmetry.C Species.C Wavefunction.C SlaterDet.C \
++				EnergyFunctional.C SampleStepper.C Basis.C FourierTransform.C Matrix.C Context.C \
++				sinft.C spline.C UnitCell.C StructureFactor.C ChargeDensity.C UserInterface.C RunCmd.C \
++				LoadCmd.C SaveCmd.C SavesysCmd.C SavedenCmd.C SpeciesCmd.C SpeciesReader.C \
++				SpeciesHandler.C XCPotential.C LDAFunctional.C PBEFunctional.C PBESolFunctional.C \
++				PBERevFunctional.C BLYPFunctional.C NonLocalPotential.C SampleReader.C SampleWriter.C \
++				StructuredDocumentHandler.C SampleHandler.C AtomSetHandler.C WavefunctionHandler.C \
+       XMLGFPreprocessor.C Base64Transcoder.C CPSampleStepper.C BOSampleStepper.C ParOptCmd.C \
+       ParallelOptimizer.C WavefunctionStepper.C SDWavefunctionStepper.C \
+       MDWavefunctionStepper.C SDIonicStepper.C MDIonicStepper.C BMDIonicStepper.C \
+@@ -41,9 +73,9 @@
+ QBOBJS = $(QBSRC:%.C=$(OBJDIR)/%.o)
+ KPTOBJS = $(KPTSRC:%.C=$(OBJDIR)/%.o)
+ SIMDOBJS = $(SIMDSRC:%.c=$(OBJDIR)/%.o)
+-ifdef CFLAGS
++#ifdef CFLAGS
```

Comment:
you won't need to change this if you don't comment out the `CFLAGS` definition above?